ruby/app/models/Citadel.rb

```diff
@@ -0,0 +1,45 @@
+require 'net/http'
+require 'json'
+
+class Citadel
```

Comment:
You can store settings here like this:
`class_attribute :product_type`

Reply:
corrected as per suggestions